azext_iot/sdk/iothub/service/models/job_request_py3.py

```diff
@@ -15,37 +15,35 @@
 class JobRequest(Model):
     """JobRequest.
 
-    :param job_id: Job identifier
+    :param job_id: The unique identifier of the job.
     :type job_id: str
-    :param type: Required.
-     The type of job to execute. Possible values include: 'unknown', 'export',
+    :param type: The job type. Possible values include: 'unknown', 'export',
      'import', 'backup', 'readDeviceProperties', 'writeDeviceProperties',
      'updateDeviceConfiguration', 'rebootDevice', 'factoryResetDevice',
      'firmwareUpdate', 'scheduleDeviceMethod', 'scheduleUpdateTwin',
      'restoreFromBackup', 'failoverDataCopy'
     :type type: str or ~service.models.enum
-    :param cloud_to_device_method: Required if jobType is cloudToDeviceMethod.
-     The method type and parameters.
+    :param cloud_to_device_method: The method type and parameters. This is
+     required if the job type is cloudToDeviceMethod.
     :type cloud_to_device_method: ~service.models.CloudToDeviceMethod
     :param update_twin:
     :type update_twin: ~service.models.Twin
-    :param query_condition: Required if jobType is updateTwin or
-     cloudToDeviceMethod.
-     Condition for device query to get devices to execute the job on
+    :param query_condition: The condition for devices to execute the job. This
+     is required if the job type is updateTwin or cloudToDeviceMethod.
     :type query_condition: str
-    :param start_time: ISO 8601 date time to start the job
+    :param start_time: The start date and time of the job in ISO 8601
+     standard.
     :type start_time: datetime
-    :param max_execution_time_in_seconds: Max execution time in secounds (ttl
-     duration)
+    :param max_execution_time_in_seconds: The maximum execution time in
+     secounds.
     :type max_execution_time_in_seconds: long
     """
 
-    # @digimaun - altered update_twin type from Twin to {object}
```

Comment:
Anything that is tagged like this `@digimaun` there was a reason for making the change, so we need to ensure that reason is gone before removing the customization.

Reply:
this is exactly why the tests are failing. I am adding them back, as I didn't realize the hub's operation has impact other than hub.py earlier.